uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/sub-controllers/leave-request-ctrl.js

```diff
@@ -10,7 +10,6 @@ define([
 
       var parentRequestCtrl = $controller('RequestCtrl'),
         vm = Object.create(parentRequestCtrl);
-
```

Comment:
Please keep the line break which separates local variables from public variables.